skipper.go

```diff
@@ -909,11 +909,12 @@ type Options struct {
 	// filters.
 	LuaSources []string
 
-	EnableOpenPolicyAgent          bool
-	OpenPolicyAgentConfigTemplate  string
-	OpenPolicyAgentEnvoyMetadata   string
-	OpenPolicyAgentCleanerInterval time.Duration
-	OpenPolicyAgentMaxBodySize     uint64
+	EnableOpenPolicyAgent             bool
+	OpenPolicyAgentConfigTemplate     string
+	OpenPolicyAgentEnvoyMetadata      string
+	OpenPolicyAgentCleanerInterval    time.Duration
+	OpenPolicyAgentMaxRequestBodySize int64
```

Comment:
Maybe:
OpenPolicyAgentMaxRequestBodySize -> OpenPolicyAgentMaxConcurrentRequestsForBodyFilter 
I wonder a bit about what this semaphore has to do with the body size other than the filter does body inspection.

Reply:
Maybe: 
OpenPolicyAgentMaxPerRequestBodySize
and OpenPolicyAgentMaxConcurrentRequestBodyMemory

but also open to other suggestions. One effectively controls how much memory each request can consume and the second is a shared constrained resource (the total buffer that can be used to parse bodies) which is protected by the semaphore.

Reply:
then let's not change it, I think I just overlooked the semaphore use